app/src/pages/Run/RunPanel/RunTimer.tsx

```diff
@@ -1,23 +1,48 @@
 import * as React from 'react'
-
+import { useSelector, useStore } from 'react-redux'
+import padStart from 'lodash/padStart'
+import { useInterval } from '@opentrons/components'
 import styles from './styles.css'
+import {
+  getRunSeconds,
+  getPausedSeconds,
+  getStartTimeMs,
+} from '../../../redux/robot/selectors'
+import { format } from 'date-fns'
 
-export interface RunTimerProps {
-  startTime: string | null
-  runTime: string
+function formatSeconds(runSeconds: number): string {
+  const hours = padStart(`${Math.floor(runSeconds / 3600)}`, 2, '0')
+  const minutes = padStart(`${Math.floor(runSeconds / 60) % 60}`, 2, '0')
+  const seconds = padStart(`${runSeconds % 60}`, 2, '0')
+  return `${hours}:${minutes}:${seconds}`
 }
 
-export function RunTimer(props: RunTimerProps): JSX.Element {
-  const { startTime, runTime } = props
-  const startTimeStamp = startTime ?? ''
+export function RunTimer(): JSX.Element {
+  const [now, setNow] = React.useState(Date.now())
+  const state = useStore().getState()
+  const startTimeMs = useSelector(getStartTimeMs)
+  const runSeconds = getRunSeconds(state, now)
+  const pausedSeconds = getPausedSeconds(state, now)
```

Comment:
Even if a given selector function may not be a [reselect](https://github.com/reduxjs/reselect) `createSelector` selector, you should still use `useSelector` over `useStore`:

```suggestion
  const runSeconds = useSelector(state => getRunSeconds(state, now))
  const pausedSeconds = useSelector(state => getPausedSeconds(state, now))
```